src/protocol/ascii.rs

```diff
@@ -68,9 +66,7 @@ impl AsciiProtocol<Stream> {
         );
         if command == StoreCommand::Cas {
             if options.cas.is_none() {
-                return Err(MemcacheError::ClientError(String::from(
-                    "cas command should have a casid",
-                )));
+                panic!("cas value should be present");
```

Comment:
Same as above mentioned.

Reply:
For cas command to work, there should be a cas value. If it wasn't passed, it means the client implementation is horribly wrong.

Also, there are tests for this. So, we should be fine.